src/core.js

```diff
@@ -114,10 +114,21 @@ function date(format) {
       timestamp = d.getTime();
     }
     var result = '';
+    var grab_next = false;
     for (var i = 0; i < format.length; i++) {
       var character = format.charAt(i);
+      if (grab_next) {
+          result += character;
+          grab_next = false;
+          continue;
+      }
       switch (character) {
 
+        case '\\':
+          grab_next = true;
+          console.log(grab_next);
+          continue;
+
```

Comment:
@jacobembree This is the only snippet I am concerned/confused about. Please let me know when you have a chance, thanks.